### DISCUSSION TITLE
Practica 2

### DISCUSSION THREAD

**Discussion by JuanSposada** [Launch-X-Latam/MisionFrontEnd#219] [Show and tell]
Compañeros les dejo mi practica 2 de html Tacos Rocket. Agradecería su feedback. Saludos
https://github.com/JuanSposada/launch-x-innovaccion-virtual/tree/main/MISION%20FRONT%20END/PRACTICA%202

**Comment by doguedogue** [comment]
Genial, por qué no la publicas con **github pages** para que te de una URL y puedas mostrarla más fácilmente 🚀

**Reply by JuanSposada** [reply]
No sabia que se podía. Gracias compañero

**Comment by EsthebanDiego** [comment]
Esta muy bien, yo solo le cambiaria la etiqueta del section en la direccion a una etiqueta address, para usar un poco mas de etiquetas semanticas

**Comment by floressdi** [comment]
Me encanto tu logo
